include/crm/pengine/internal.h

```diff
@@ -529,6 +529,7 @@ void pe_action_set_flag_reason(const char *function, long line, pe_action_t *act
 
 void pe__set_resource_flags_recursive(pe_resource_t *rsc, uint64_t flags);
 void pe__clear_resource_flags_recursive(pe_resource_t *rsc, uint64_t flags);
+void pe__clear_flag(pe_working_set_t *data_set, uint64_t flag);
```

Comment:
Data sets have multiple flag types (resource flags, find flags, data set flags, node flags, graph flags, action flags, ordering flags, ...) so let's indicate somehow that this is for resource flags for all resources in the data set.